manifests/mod/passenger.pp

```diff
@@ -343,6 +346,7 @@
   $passenger_installed_version                                                               = undef,
   $passenger_instance_registry_dir                                                           = undef,
   $passenger_load_shell_envvars                                                              = undef,
+  Optional[Boolean] $passenger_preload_bundler                                               = undef,
```

Comment:
Wouldn't false be the better default?
```suggestion
  Boolean $passenger_preload_bundler                                                         = false,
```

Reply:
No. Right now `undef` means "don't output it at all" and the boolean translates to `On`/`Off`. This is the correct behavior IMHO, especially since older Passenger versions don't support this so it'd fail on an unexpected statement.